ansible/playbooks/sap-hana-cluster.yaml

```diff
@@ -7,10 +7,12 @@
     # is_primary is selected so that tasks that need to be issued one are honoured correctly
     is_primary: "{{ ansible_play_hosts[0] == inventory_hostname }}"
     primary_hostname: "{{ ansible_play_hosts[0] }}"
-    use_sbd: yes
+    use_sbd: "{{ default('yes') }}"
```

Comment:
It is just a default value in case user does not specify anything in ansible section. Hard-coded value would always deploy SBD fencing in cluster config.

Reply:
in a next PR we can think about populating this one from Terraform:
- https://github.com/SUSE/qe-sap-deployment/blob/main/terraform/azure/outputs.tf and https://github.com/SUSE/qe-sap-deployment/blob/main/terraform/azure/inventory.tmpl changed to read `use_sbd` and set a variable in the inventory.yaml
- ansible/playbooks/sap-hana-cluster.yaml maybe do not need any change to read that value from the inventory

Similar approach of what we did in https://github.com/SUSE/qe-sap-deployment/pull/133

Reply:
Just for the record: 
- I rebased this PR and use_sbd variable is now taken from the inventory file. 
- I did here the same for variables resource_group_name, subscription_id, tenant_id